### PR TITLE
fixed some UI components for oxides and elements in edit chemical analyses page

### PR DESCRIPTION
**Pull request by adt123**

### CODE REVIEW

**Comment by tamblc** [comment]
I merged some changes into interface that conflict with your changes, if you can rebase your branch off of the master branch, it should resolve those merge conflicts.

**Comment by TheoBr** [comment]
As mentioned by @tamblc , some of these things were fixed with pull request https://github.com/metpetdb/interface_v2/pull/6 . To solve the merge conflict that's occurring, you need to either rebate or merge off of master. I recommend rebasing as it will make the diffs much cleaner. 

Very good writeup on merging and rebasing from master here: https://www.atlassian.com/git/tutorials/merging-vs-rebasing/conceptual-overview

**Comment by TheoBr** [comment]
@adt123 I can handle the conflict if you like, I'm updating the versioning on this repository anyways

File: app.py [hunk 1]
```diff
@@ -337,6 +337,7 @@ def edit_subsample(id):
     if new:
         sample = get(env("API_HOST")+"samples/"+request.args.get("sample_id")+"/", params = {"fields": "id,number,owner"}, headers = headers).json()
     subsample = dict(request.form)
+    print subsample
```
Comment (TheoBr) [inline app.py]: Is this print necessary?

**Pull request merged**